docs/guides/admin/docs/configuration/elasticsearch.md

```diff
@@ -0,0 +1,35 @@
+Elasticsearch Configuration
+===========================
+
+Elasticsearch powers the external API as well as the administrative user interface of Opencast.
+
+Running Elasticsearch
+---------------------
+
+When running Elasticsearch, it is mandatory to deploy the same major version as the client library used by Opencast has,
+since the client commands will otherwise not match the server. To check the version, take a look at
+[the maven dependency declaration for the elasticsearch bundle in the search module](https://github.com/opencast/opencast/blob/develop/modules/search/pom.xml).
+
+For example, to quickly spin up an external Elasticsearch matching the current version using Docker, run
```

Comment:
The more I read this the more I think the whole section should be removed.  Installation documentation should take care of *installing* ES, whether that's via Docker or via packages.  Unless an adopter is *very* advanced they shouldn't be trying to figure out which version of ES they need...

Reply:
If it's necessary to address the version, then definitely note it in the install doc. But otherwise, I agree with the sentiment that it can be removed from this page entirely.